optscale_arcee/arcee.py

```diff
@@ -52,8 +52,9 @@ def __init__(
         self.sender = Sender(endpoint_url, ssl, self.shutdown_flag)
         self.hb = None
         self._run = None
-        self._tags = {}
+        self._tags = dict()
         self._name = None
+        self._hyperparams = dict()
```

Comment:
Just wondering if there is a reason for this way of initiation, but `{}` works "faster" and "shorter"

https://switowski.com/blog/dict-function-vs-literal-syntax/#:~:text=When%20we%20call%20%7B%7D%20%2C%20we,overriding%20the%20dict()%20function.

Reply:
just a habit

Reply:
Thanks for the article, I'll take